upgrade/1.2/scripts/upgrade/ncn-upgrade-ceph.sh

```diff
@@ -28,7 +28,7 @@ state_recorded=$(is_state_recorded "${state_name}" ${upgrade_ncn})
 if [[ $state_recorded == "0" ]]; then
```

Comment:
We will not be running this.  If we do decide to push for the upgrade then this will be re-written as most of the upgrade script will no longer be valid.